src/index/index_construct.c

```diff
@@ -58,9 +58,7 @@ static void _Index_PopulateNodeIndex
 		//----------------------------------------------------------------------
 
 		GrB_Info info;
-		info = Delta_MatrixTupleIter_attach(&it, m);
```

Comment:
yes much better because in the attach it can search from which row to start and it can take a lot of time and it is a waste if we give the range already